src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,59 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private static Connection connection;
+
+    private static ConnectionService INSTANCE;
+
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+
+    private static final String JDBC_URL = "jdbc:postgresql://developers-db:5432/";
```

Comment:
Split this URL into following constants:
- schema
- host
- port

Reply:
fixed